microsoft-edge/devtools-guide-chromium/issues/index.md

```diff
@@ -43,7 +43,7 @@ Feedback in the **Issues** tool is provided by several sources, including the Ch
 
 1. Go to a webpage that contains issues to fix.  For example, open the [accessibility-testing demo page](https://microsoftedge.github.io/Demos/devtools-a11y-testing/) in a new tab or window.
 
-1. Open DevTools.  After a few seconds, the **Issues counter** (![Issues counter.](../media/issues-counter-icon.msft.png)) appears in the upper right corner of DevTools.
+1. Open DevTools.  After a few seconds, the **Issues icon** (![Issues icon.](media/issues-icon.png)) appears in the upper right corner of DevTools.
```

Comment:
The icon moved, it's no longer in the top right corner. So we should update the wording.
Here is a suggestion:

```suggestion
1. Open DevTools.  After a few seconds, a blue information badge appears on the **Issues icon** in the **Activity bar** (![Issues icon.](media/issues-icon.png)).
```